addon/hint/show-hint.js

```diff
@@ -200,7 +200,8 @@
     var widget = this, cm = completion.cm;
 
     var hints = this.hints = document.createElement("ul");
-    hints.className = "CodeMirror-hints";
+    var theme = completion.cm.options.theme;
+    hints.className = `CodeMirror-hints ${theme}`;
```

Comment:
The addons should be ES5, so you can't use template literals here.

Reply:
@marijnh okay. I'll change to 
`var theme = "CodeMirror-Hints " + theme;`